scaffolder-templates/gitlab-merge-request/template.yaml

```diff
@@ -3,7 +3,7 @@ kind: Template
 metadata:
   name: gitlab-merge-request
   title: GitLab Merge Request Action template
-  description: scaffolder v1beta2 template demo publishing to MR on existing git repository
```

Comment:
Think you can just drop the version from this description to be honest

Reply:
👍